verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -5,6 +5,10 @@ describe('Linelist table', function () {
         cy.login();
     });
 
+    afterEach(() => {
```

Comment:
Is this needed? I don't see anything being seeded in these tests.

Reply:
addCase does seed a location.